test/client/components/ConnectedPostList.test.js

```diff
@@ -0,0 +1,47 @@
+require('../support')
+import { mocks, helpers } from '../../support'
+import ConnectedPostList from '../../../src/containers/ConnectedPostList'
+import {
+  findRenderedDOMComponentWithClass,
+  renderIntoDocument
+} from 'react-addons-test-utils'
+
+describe('ConnectedPostList', () => {
+  describe('with freshCount', () => {
+    let community = {id: '1', name: 'Foomunity', slug: 'foo'}
+
+    let state = {
+      communities: {
+        [community.slug]: community
+      },
+      people: {
+        current: {
+          id: 'x',
+          name: 'Mr. X',
+          avatar_url: '/img/mrx.png'
+        }
+      },
+      posts: {1: {
+        id: 1, name: 'post', description: 'ra', type: 'offer',
+        user: {id: 1, name: 'jo', avatar_url: ''},
+        communities: [community]
+      }},
+      totalPostsByQuery: {'subject=community&id=foo': 1},
+      postsByQuery: {'subject=community&id=foo': [1]},
+      pending: {},
+      countFreshPostsByQuery: {
+        'subject=community&id=foo': 3
+      }
+    }
+
+    it('displays new post notification', () => {
+      const props = {subject: 'community', id: 'foo'}
+      const store = mocks.redux.store(state)
+      const context = {store, dispatch: store.dispatch, community}
```

Comment:
you don't need to have `dispatch` here unless you want a custom spy; `createElement` automatically sets it